server/src/main/java/model/placement/fleet/FleetBuilder.java

```diff
@@ -0,0 +1,31 @@
+package model.placement.fleet;
+
+import model.Coordinates;
+import model.placement.ship.DirectedShip;
+import model.placement.ship.Direction;
+import model.placement.ship.PlacedShip;
+import model.placement.ship.ShipType;
+
+public class FleetBuilder {
+    private Fleet<PlacedShip> fleet;
+
+    private FleetBuilder() {
+        fleet = new Fleet<>();
+    }
+
+    public static FleetBuilder create() {
+        return new FleetBuilder();
+    }
+
+    public FleetBuilder appendShip(Coordinates headCoordinates, ShipType shipType, Direction direction) {
```

Comment:
Maybe wrapping parameters/fields ? It could be possible that there are common wrappe in some places